calcom/cal.com
# Add event type redirect URL to Stripe callback

## What does this PR do?



This PR adds the event type redirect URL if enabled to the Stripe `confirmParams`

Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?



- Create an event type with Stripe enabled and a redirect URL
- Create a booking, it should redirect you to the redirect URL
    - Do this for both charge on booking & charging no-show fee
- Go through the same booking flow again but this time disable redirect URL

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
